### PR TITLE
My Portfolio : contact section added

### PR DESCRIPTION
**Pull request by anshuman7negi**
The description followed during this branch -
Create a new branch according to the GitHub flow rules.
I updated the index.html file by adding a contact section.
Updated the style.css file by adding CSS related to the contact section,

### CODE REVIEW

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @anshuman7negi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 

You have submitted a project, that is not in line with the project requirement on work and submission as shown in the image below
com
- [ ] You and your partners are required to have equal commits in this repo before submission and currently the image below only shows commits from you. please kindly alert your partner about this so he can add his commits before resubmitting
comm


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -243,7 +248,25 @@ <h3>LET’S CONNECT</h3>
...
-
+       <!--Start of Contact Page-->
+       <section class="contact-page">
+        <div class="contact-block">
+            <div class="description">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                    I’d love to help with it!</p>
+            </div>
+            <form action="#">
```
Comment (LionRouge1) [inline index.html]: - [ ] Nice work implementing the form, it look really good :clap: However it states in the project requirement that data should be collected by using the [Formspree service](https://formspree.io/html). Therefore I suggest you:
  - add the `action` attribute to your form tag and its value should be similar to `https://formspree.io/f/{form_id}` where `{form_id}`  is `id` giving you when create an account on [formspree service](https://formspree.io/html).
  - add the `method` attribute to your `form` tag and its value  equals `POST`

File: index.html [hunk 2]
```diff
@@ -243,7 +248,25 @@ <h3>LET’S CONNECT</h3>
...
+                <fieldset>
+                    <input type="text" placeholder="Lorem Ipsum">
```
Comment (LionRouge1) [inline index.html]: - [ ] It required that the name field accepts **up to 30 chars**, therefore kindly add `maxlength="30"` attribute to the name field.
- [ ] It required that user shouldn't be able to submit a form when one or more fields is empty, Therefore kindly add `required` attribute to the name fields.

File: index.html [hunk 3]
```diff
@@ -243,7 +248,25 @@ <h3>LET’S CONNECT</h3>
...
+                    <input type="email" placeholder="loremipsum@gmail.com">
```
Comment (LionRouge1) [inline index.html]: - [ ] Same for the email input, kindly add `required` attribute to the input.

File: index.html [hunk 4]
```diff
@@ -243,7 +248,25 @@ <h3>LET’S CONNECT</h3>
...
+                    <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here..."></textarea>
+                </fieldset>
```
Comment (LionRouge1) [inline index.html]: - [ ] It's requested message fields accepts **up to 500 chars**, Therefore I suggest you add `maxlength="500"` attribute to the message input.
- [ ] As pointed earlier :point_up: Kindly add `required` attribute to this `textarea`

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @anshuman7negi & @Lawmsangi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job implementing the form it looks almost identical to the Figma design :1st_place_medal: 
- Nice job using **flexbox** to lay out the form elements :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @anshuman7negi , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Form  validation applied
- [ ] All requested changes implemented ✔️ 
- [ ] Good readme
 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**